### ISSUE TITLE
Refreshing 'My Favorites' Page

### ISSUE THREAD

**Issue by TheTopher** [ThatConference/that.us#188]
**Is your feature request related to a problem? Please describe.**
When I select "My Favorites" from my User-Bug, I am put on my "My Favorites" page (expected).  However, the UI does update if I unfavorite the session from the list.  It would be nice if I could refresh the page by reselecting "My Favorites" again or have a refresh button.

**Describe the solution you'd like**
Allow update through reselecting "My Favorites" again or have a refresh button.

**Describe alternatives you've considered**
Above

**Additional context**
N/A

**Comment by csell5** [comment]
@TheTopher by "not update".. do you mean pull the card off the screen?  

ref: #64

**Comment by TheTopher** [comment]
Yes, that is correct.  The card remains on the screen after I unfavorite it.

**Comment by TheTopher** [comment]
As a note, I can see value in keeping an unfavorite'd item on the page (e.g. maybe I unfavorited the wrong one).  However, there should be a way to refresh (even if it is deliberate) the page without leaving the page.
